client/app/components/sections/Topbar/Topbar.js

```diff
@@ -22,14 +22,27 @@ import MenuMobile from '../../composites/MenuMobile/MenuMobile';
 import MenuPriority from '../../composites/MenuPriority/MenuPriority';
 import SearchBar from '../../composites/SearchBar/SearchBar';
 
-const profileDropdownActions = function profileDropdownActions(routes, username) {
+
+const LABEL_TYPE_MENU = 'menu';
+const LABEL_TYPE_DROPDOWN = 'dropdown';
+const SEARCH_PARAMS_TO_KEEP = ['view', 'locale'];
+const SEARCH_PARAMS = ['q', 'lq'];
+const DEFAULT_CONTEXT = {
+  marketplace_color1: styleVariables['--customColorFallback'],
+  marketplace_color2: styleVariables['--customColor2Fallback'],
```

Comment:
Wasn't the customColor2 removed in the new layout? It shouldn't affect anything anymore? The only place where it was used was the Post a new listing button color and even the option to configure it is removed if the new topbar is enabled.